### PR TITLE
bc1qx8k229gazptgfpfctkc37vw0wwz3anuc243w22

### PR DESCRIPTION
**Pull request by Rja91**
Ty

### CODE REVIEW

File: pools.json [hunk 1]
```diff
@@ -1234,5 +1262,143 @@
             "name": "MARA Pool",
             "link": "https://marapool.com"
         }
+    },
+    "slugs": {
```
Comment (jiangjinyuan) [inline pools.json:1266]: not support this filed "slugs".

File: README.md [hunk 2]
```diff
@@ -1,6 +1,5 @@
-Blockchain Pools
-======================
+# Bitcoin Mining Pools
 
-Bitcoin Mining Known Pools Tracking Tags for https://btc.com/stats/pool
+Coinbase Tags used on https://mempool.space/mining/pools
```
Comment (jiangjinyuan) [inline README.md:3]: We don't want to update this file.

**Comment by jiangjinyuan** [comment]
@Rja91 Pls resolve the conflicts.

**Comment by wiz** [comment]
this PR is just spam by some random person, with commits from our fork at @mempool/mining-pools - please close this PR

**Comment by jiangjinyuan** [comment]
> this PR is just spam by some random person, with commits from our fork at @mempool/mining-pools - please close this PR

Ok, I will close the PR!

**Comment by jiangjinyuan** [comment]
Closed!

**Pull request closed without merging**